selfdrive/ui/ui.cc

```diff
@@ -189,13 +189,30 @@ static void update_state(UIState *s) {
     scene.light_sensor = std::clamp<float>(1.0 - (ev / max_ev), 0.0, 1.0);
   }
   scene.started = sm["deviceState"].getDeviceState().getStarted() && scene.ignition;
+  if (sm.updated("dragonConf")) {
```

Comment:
I moved these out of priv, I dont see any issue exposing these here since they are only setting variables :P